src/server/run_mir.cpp

```diff
@@ -162,6 +163,12 @@ extern "C" [[noreturn]] void fatal_signal_cleanup(int sig, siginfo_t* info, void
         constexpr size_t len = std::char_traits<char>::length(warning);
         [[maybe_unused]]auto n = write(STDERR_FILENO, warning, len);
     }
+    char security_log[] = "{\"datetime\": \"YYYY-MM-DDThh:mm:ssZ\", \"appid\": \"mir.canonical.com\", \"event\": \"sys_crash\", \"level\": \"WARN\", \"description\": \"Mir fatal signal received\" }\n";
```

Comment:
Is this appid correct? I would expect it to reflect the current compositor (e.g. miral-shell or Miriway).

Reply:
You're not adding the signal to the buffer? (`strncpy()` is async signal safe)